endpoint_route_handler/models/ir_http.py

```diff
@@ -71,6 +72,10 @@ def routing_map(cls, key=None):
                 cls._routing_map = {}
                 cls._rewrite_len = {}
                 cls._endpoint_route_last_version = last_version
+        finally:
+            cr.rollback()
```

Comment:
can you add a comment here pls?
Mention that the new cr is only to bypassing transaction isolation during a read  or smth similar.

Reply:
Done